### ISSUE TITLE
jsx-indent interacts badly with indent

### ISSUE THREAD

**Issue by Wilfred** [yannickcr/eslint-plugin-react#899]
Given the code:

```js
function foo() {
  const items = [
    <span>foo</span>,
    "bar",
  ];

  return <div>{items}</div>;
}
```

And the following eslintrc.json file:

```json
{
    "parserOptions": {
        "ecmaVersion": 6,
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "rules": {
        "indent": ["warn", 2],
        "react/jsx-indent": "warn"
    },
    "plugins": [
        "react"
    ]
}
```

I get the following error:

```
Expected indentation of 6 space characters but found 4  react/jsx-indent
```

The problem here is that both `indent` and `react/jsx-indent` have opinions on how this line should be indented, and they disagree.

**Comment by ljharb** [comment]
`indent` is correct here, and I think d0dfc075af92f43ebe25f4bde0fd78820ed295d8 might fix it?

**Comment by Wilfred** [comment]
I don't have a box to check the latest dev version on currently, but I agree this looks a lot like a duplicate.

**Comment by yannickcr** [comment]
The fix should be published in the coming hour. Please reopen #898 if you still encounter the problem with `v6.4.1`.
